README.rst

```diff
@@ -41,7 +41,11 @@ Some noteworthy flags::
 
     codespell -w, --write-changes
 
-The ``-w`` flag will actually implement the changes recommended by codespell. Not running with ``-w`` flag is the same as with doing a dry run. It is recommended to run this with the ``-i`` or ``--interactive`` flag.::
+The ``-w`` flag will actually implement the changes recommended by codespell. Not running with ``-w`` flag is the same as with doing a dry run. It is recommended to run this with the ``-i`` or ``--interactive`` flag.
```

Comment:
Currently we've generally got
--foo
Explanation of foo.

You've added some comments above the flag too. I would suggest for consistency/clarity either we should move this below too, or move ALL the explanations above the flags. Afterwards certainly seems to match the man pages I've looked at.

Reply:
I'd agree it's a bit out of place. I went for above, just because it applies to both `-I` and `-L`.

I guess the obvious options are to either duplicate it for both, or maybe just move it below the `-L` entry. In that case the example `wrod` could be changed to `wrod1`, which might be a little more obvious?

Reply:
Maybe for the readme particularly, we should rewrite it as:

> **Ignoring Words**
> When ignoring false positives, note that spelling errors are *case-insensitive* but words to ignore are *case-sensitive*. For example, the dictionary entry ``wrod`` will also match the typo ``Wrod``, but to ignore it you must pass ``wrod``.
> 
> The words to be ignore can be passed in two ways:
> -I
> A file with a word per line to ignore.
> -L
> A comma separated list of words to ignore on the command line.

Or something like that? I think there's a similar case with something else further down.

Reply:
Not a bad idea, and leaves it open to add more help if necessary. I sort of back-pedaled on the earlier changes, then added a section a bit farther down with internal links.

Let me know what you think :)